didyoumean/didyoumean_api_tests.py

```diff
@@ -1,13 +1,22 @@
 # -*- coding: utf-8
 """Unit tests for didyoumean APIs."""
-from didyoumean_api import didyoumean_decorator, didyoumean_contextmanager,\
-    didyoumean_postmortem, didyoumean_enablehook, didyoumean_disablehook
-from didyoumean_common_tests import TestWithStringFunction,\
-    get_exception, no_exception, NoFileIoError, unittest_module
 import contextlib
 import sys
 import os
 
+from .didyoumean_api import (
+    didyoumean_decorator,
+    didyoumean_contextmanager,
+    didyoumean_postmortem,
+    didyoumean_enablehook,
+    didyoumean_disablehook)
+from .didyoumean_common_tests import (
+    TestWithStringFunction,
+    get_exception,
+    no_exception,
+    NoFileIoError,
+    unittest_module
+    )
 
```

Comment:
PEP8 complains that:
didyoumean/didyoumean_api_tests.py:21:1: E302 expected 2 blank lines, found 1